client/my-sites/themes/theme-showcase.jsx

```diff
@@ -159,7 +159,10 @@ class ThemeShowcase extends Component {
 
 	scrollToSearchInput = () => {
 		if ( ! this.props.loggedOutComponent && this.scrollRef && this.scrollRef.current ) {
+			const yOffset = -55; // A number that takes the Adminbar as well as the screen options into account on mobile view ports.
 			this.scrollRef.current.scrollIntoView();
+			const y = this.scrollRef.current.getBoundingClientRect().top + window.pageYOffset + yOffset;
+			window.scrollTo( { top: y } );
```

Comment:
Could we do this only for mobile? It feels odd that we're scrolling for desktop too, when there's plenty of space for everything.